app/views/works/_form.html.erb

```diff
@@ -44,7 +44,14 @@
       <% if !@wizard_mode %>
         <hr />
         <div class="deposit-uploads">
-          <%= render(partial: 'works/uploads_form', locals: { form: form }) %>
+          <% if @work.approved? %>
+            <section class="uploads">
+              <h2 class="h2"><%= t('works.uploads.post_curation.heading') %></h2>
+              <%= render(partial: 'works/uploads/post_curation_s3_resources') %>
```

Comment:
I think this is the only place this partial is used: Maybe move the h2 into it for consistency?

Reply:
This view is utilized a couple places.  I grabbed it from the show.